media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -686,7 +686,14 @@ VAStatus MediaLibvaCaps::CreateEncAttributes(
 
         if (IsHevcProfile(profile))
         {
-            attrib.value |= VA_RC_ICQ | VA_RC_VCM;
+          if (entrypoint != VAEntrypointEncSliceLP)
```

Comment:
I think the following approach is more straightforward.
```
attrib.value |= VA_RC_ICQ | VA_RC_VCM;
if (entrypoint != VAEntrypointEncSliceLP)
 {
      attrib.value |= VA_RC_QVBR;
 }
```

Reply:
updated